### ISSUE TITLE
Add minutes to model

### ISSUE THREAD

**Issue by ayyubibrahimi** [eye-on-surveillance/the-great-inquirer#21]
- In addition to a guide re how to query what was recently up for vote/outcome of the vote (Ethan reviews agendas to know when something new is coming out)
- [Minutes](https://council.nola.gov/meetings/)

**Comment by ayyubibrahimi** [comment]
@marvinmarnold can you expand on what data Ethan collects? I was going to close this because minutes and agendas have been added.

**Comment by marvinmarnold** [comment]
tldr: I think we are good for now with the minutes/agendas

I'll follow up with them once we start sharing out. The other major thing I think he specifically looks out for is "first readings". This is when new proposed ordinances are formerly introduced at the end of council meetings. There is no discussion about these items, they just need to be formally read into record as the first step. Then the soonest they can go up for vote is the next council meeting. These items aren't always reliably recorded in minutes/agendas though.

I don't think they are very relevant at this point. But if we do build out functionality highlighting the most recent council meeting, that would be really important to highlight.

Beyond that, I imagine he's just also just listening for keywords like "music", "venue", "french quarter", etc. I can imagine a google alerts like service where users register keywords they care about and get notifications whenever they come up in councilmeetings.
